jenkins/scripts/integration_delete.sh

```diff
@@ -28,6 +29,17 @@ then
 fi
 echo "Running in region: $OS_REGION_NAME"
 
+if [[ "$OS_REGION_NAME" != "Kna1" ]]
+then
+  # Find executer floating ip
+  TEST_EXECUTER_FIP_ID="$(openstack floating ip list --tags "${TEST_EXECUTER_FIP_TAG}" -f json | jq -r '.[].ID')"
```

Comment:
Theoretically we could get more than one IP here. Should we loop over them and delete instead of assuming we only get one?

Reply:
You could also pipe to `xargs`.

Reply:
Today I learned about `xargs` :smile: 
Excellent idea!